custom_components/plugwise/translations/nl.json

```diff
@@ -259,7 +259,7 @@
         "name": "Huishoudelijk warmwater instelling"
       },
       "maximum_boiler_temperature": {
-        "name": "Maximum boiler temperatuurinstelling"
+        "name": "Maximum boilertemperatuur instelling"
```

Comment:
Deze zou ik zker omdraaien Instelling maximale boilertemperatuur (of instelling maximale temperatuur boiler)